pkg/networkmanager/index.html

```diff
@@ -284,17 +284,17 @@ <h2 class="panel-title" translatable="yes">Unmanaged Interfaces</h2>
         </div>
       </div>
 
-      <label class="control-label" translatable="yes">Mode</label>
+      <label class="control-label" for="network-bond-settings-mode-select" translatable="yes">Mode</label>
       <div id="network-bond-settings-mode-select">
```

Comment:
You now create the target IDs dynamically for the actual button, so they need to get dropped from this containing div -- otherwise you have duplicate IDs.

Reply:
no, those divs are replaced by a specific component not inserted into it.
I know it is a bit confusing that we define the same ID twice. But it is used once to target the correct div and replace it and second time so we can can reference label to it. (and yes, first version of this PR did not work at all :D)